graphql-jpa-query-schema/src/main/java/com/introproventures/graphql/jpa/query/schema/impl/QraphQLJpaBaseDataFetcher.java

```diff
@@ -900,7 +893,7 @@ protected final DataFetchingEnvironment wherePredicateEnvironment(DataFetchingEn
     @SuppressWarnings( { "unchecked", "rawtypes" } )
     protected Object convertValue(DataFetchingEnvironment environment, Argument argument, Value value) {
         if (value instanceof NullValue) {
-            return null;
+            return value;
```

Comment:
@negesti What is the reason to change `return null` to `return value`? I also realized I do not have test coverage for it. This should convert GraphQL AST `NullValue` to Java `null`.